guardian/recipes
# Feature: Image Picker Component

## What does this change?
Add an Image picker component to select the correct picture.

## How to test
Run locally and see whether pictures are displayed on the RHS. Do they make sense? Can you select the correct one?

## Image
![image](https://user-images.githubusercontent.com/64908337/110625484-63d6d300-8197-11eb-9ae5-d60a29156471.png)
